src/module/rules/rule-element/item-alteration/alteration.ts

```diff
@@ -21,6 +21,11 @@ class ItemAlteration extends foundry.abstract.DataModel<RuleElementPF2e, ItemAlt
                 initial: undefined,
                 choices: R.keys(ITEM_ALTERATION_HANDLERS),
             }),
+            fromEquipment: new fields.BooleanField({
+                required: true,
+                nullable: false,
+                initial: true,
```

Comment:
```suggestion
                initial: false,
```

Reply:
One thought I had was having the alteration default to false and the rule element default to true, but I wasn't sure about that.

Reply:
I don't think it will cause them to behave differently since this is the ItemAlteration RE

Reply:
It will cause item alterations to behave differently from flat modifiers, if I structure it less vaguely.

Reply:
Done. Programmatically right now, the checks are superfluous (fromEquipment has no bearing on other types of alterations anyways). But they're added so that at least internally it will stay consistent. This does add some degree of hardcoding though, that would be better of leaving entirely to the handler.